config/initializers/controllers.rb

```diff
@@ -7,8 +7,6 @@
   helper ApplicationHelper
   helper OpenStax::Utilities::OsuHelper
 
-  rescue_from Exception, :with => :rescue_from_exception
```

Comment:
put `use_openstax_exception_rescue` here (get application controller and api controller in one shot)